pkgs/applications/misc/protonup-qt/default.nix

```diff
@@ -0,0 +1,32 @@
+{ appimageTools, fetchurl, lib }:
+let
+  pname = "protonup-qt-bin";
+  name = "protonup-qt";
```

Comment:
```suggestion
  pname = "protonup-qt";
```

Reply:
When using the appimage wrapper, the resulting binary name is set to `name`. This change will make the output binary `$out/bin/protonup-qt-2.7.4`, which will then change every time the package is updated. I am hardcoding `name` to avoid this, would it be preferable to add a line to rename it in the `extraInstallCommands`?

Reply:
No, this is unfortunately how appImage tooling works in Nixpkgs. We ought to change that but this PR isn't the place to do so.

Reply:
> I am hardcoding `name` to avoid this, would it be preferable to add a line to rename it in the `extraInstallCommands`?

I think we are renaming the binary in every other appimage package.

See https://github.com/NixOS/nixpkgs/blob/a362cc95bd578019ab03558d7e39fadee9abfbbb/pkgs/applications/blockchains/mycrypto/default.nix#L35 (That package is still constructing name manually from pname+version which is no longer necessary.)

The problem with having a name variable here is that name is already occupied and set to ``${pname}-${version}`` hence should not be overwritten.

We should think about changing this but not in this PR.

Reply:
Got it, I'll remove the name declaration and rename it like this.

Reply:
Using  `mv $out/bin/{${pname}-${version},${pname}}`, because using `${name}` will fail because it hasn't been declared. The package you linked manually declares `name = "${pname}-${version}"` but I'd rather avoid that.